### PR TITLE
fix(chapters): Text corrections, text updates in chapter 3 and 8

### PR DESCRIPTION
**Pull request by TSNheathen**
Need to decide which version of a word to use:
JavaScript
javascript 
JavaScript (JS) - and later in text only JS?

### CODE REVIEW

File: chapters/qa-03.mdx [hunk 1]
```diff
@@ -57,28 +61,28 @@ Poměr slučovací: „Podejte nám zprávu písemně nebo telefonicky”
...
 |        1 (true, pravda)        |          1 (true, pravda)          |                1 (true, pravda)                 |                      Zprávu můžu podat oběma způsoby                       |
-|        1 (true, pravda)        |         0 (false,nepravda)         |                1 (true, pravda)                 |                       Zprávu můžu podat jen písemně                        |
-|       0 (false,nepravda)       |          1 (true, pravda)          |                1 (true, pravda)                 |                     Zprávu můžu podat jen telefonicky.                     |
-|       0 (false,nepravda)       |         0 (false,nepravda)         |               0 (false,nepravda)                | Pokud podáš zprávu jinak než písemně nebo telefonicky, tak nebude přijata. |
+|        1 (true, pravda)        |         0 (false, nepravda)         |                1 (true, pravda)                 |                       Zprávu můžu podat jen písemně                        |
```
Comment (pegak) [inline chapters/qa-03.mdx:64]: Do you have Prettier install? Formatting is now broken (at least in Github).

**Review by pegak — CHANGES_REQUESTED** [review]

**Comment by pegak** [comment]
I think that we should use JavaScript at least once in each chapter, where it is used. Then JS later in the chapter, what do you think, @xhemalov? :)

File: chapters/qa-03.mdx (hunk 1 above)
Comment (TSNheathen) [reply chapters/qa-03.mdx:64]: Preview on local is without change
image

File: chapters/qa-03.mdx [hunk 2]
```diff
@@ -101,16 +105,16 @@ Cyklům se věnuje článek [Lekce 5 - Cykly v JavaScriptu](https://www.itnetwo
...
-Pokud chceš, můžeš zkusit vymyslet, jak použít _while_ cyklus místo* for* cyklu na některém z příkladů ze cvičení s želvou pro for cyklus.
+Cyklus _for_ se používá pokud známe počet opakování. Ale co když chceme říct počítači opakuj dokud proměnná x nebude menší než nula. Takový cyklus může mít jedno, mnoho nebo žádné opakování, záleží na hodnotě proměnné. Předem nevíme kolik opakování se vykoná, takže cyklus for nepřipadá k úvahu. V takovém případě se používá cyklus _while_. **Pomocí while cyklu se dá napsat jakýkoliv cyklus, i ten pro který známe počet opakování. Jinak řečeno každý for cyklus lze zapsat jako while cyklus**, ale ne obráceně. \
+Pokud chceš, můžeš zkusit vymyslet, jak použít _while_ cyklus místo **for** cyklu na některém z příkladů ze cvičení s želvou pro for cyklus.
```
[the remark this answers is not in the document]
Comment (TSNheathen) [reply chapters/qa-03.mdx]: **for** changed to *for*

File: chapters/qa-08.mdx [hunk 3]
```diff
@@ -84,15 +87,19 @@ Mezi základní příkazy pro větvení patří například:
...
+Jakmile má vývojář svou práci hotovou je potřeba změny sloučit. K tomu slouží **Pull Request (PR)**, díky kterému dojde ke sloučení. Ale ne jen tak. Tady se liší postup firma od firmy a občas projekt od projektu.
+Jakmile má vývojář svou práci hotovou je potřeba změny sloučit. K tomu slouží **Pull Request (PR)**, díky kterému dojde ke sloučení. Ale ne jen tak. Tady se liší postup firma od firmy a občas projekt od projektu.
 
-Nejprve musí projít všechny statické analýzy kódu. Například prettier - formátuje kód podle pravidel (každý máme různé způsoby psaní a je důležité být jednotný) nebo linter (kontroluje dodržení různých pravidel - ne nutně stylistických, ale také funkčních - třeba, že má každý obrázek popisek).
+Nejprve musí projít všechny statické analýzy kódu. Například **prettier** - formátuje kód podle pravidel (každý máme různé způsoby psaní a je důležité být jednotný) nebo **linter** (kontroluje dodržení různých pravidel - ne nutně stylistických, ale také funkčních - třeba, že má každý obrázek popisek).
```
Comment (pegak) [inline chapters/qa-08.mdx:101]: Please fix doubled content.

**Review by pegak — CHANGES_REQUESTED** [review]

**Review by pegak — APPROVED** [review]

**Pull request merged**